assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -293,6 +293,7 @@ function AssignmentPlanningV2 (props) {
               <UserSettingSnackbar
                 saved={!mutationError && !mutationLoading && settingChanged}
                 response={{ default: 'success' }}
+                debounceAmount={750}
               />
```

Comment:
This is milli second i suppose? why is this number chosen?

Reply:
Not much thought went into it other than to add a buffer approximately around the duration of the other debounced input we had (originally 500ms). I ran it by @gsilver, and the 500-1000ms option seemed to be a good range. I'm also open to other guidance on this (and we can customize it later)

Reply:
I think 750 seems fine, I just want to understand the reason.